app/models/viewmodels/managementtools/Keyword.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewmodels.managementtools
+
+import models.CaseStatus.CaseStatus
+import models.{ApplicationType, CaseStatus}
+
+case class Keyword(
```

Comment:
Can you check that the model for Keyword matches the one in the backend? For reference : DIT-2760

Reply:
So this is just a stub at the moment for this ticket which will be replaced with the new Keyword model matching the backend in DIT-2755, which is in progress.